wiki/en/Tips-Tricks-More.md

```diff
@@ -84,16 +84,13 @@ Here is the script:
 
 The volume fader, pan control and mute and solo buttons in the client's mixer window strips can be controlled using a MIDI controller by using the `--ctrlmidich` parameter (note: only available for use with macOS and Linux using Jamulus version 3.7.0 or higher, and on Windows using the Jamulus version with JACK support). To enable this feature, Jamulus must be launched with `--ctrlmidich`. There is one global MIDI channel parameter (1-16) and two parameters you can set for each item controlled: `offset` and `consecutive CC numbers`. Set the first parameter to the channel you want Jamulus to listen on (0 for all channels) and then specify the items you want to control (f = volume fader; p = pan; m = mute; s = solo; o = mute myself) with the offset (CC number to start from) and number of consecutive CC numbers. There is one exception that does not require establishing consecutive CC numbers which is the "Mute Myself" command - it only requires a single CC number as it is only applied to one's own audio stream. Take the following example:
 
-`--ctrlmidich "1;f0*8;p16*8;s32*8;m48*8;o64"`
+`--ctrlmidich '1;f0*8;p16*8;s32*8;m48*8'`
```

Comment:
The whole of this section ("Using ctrlmidich for MIDI controllers") should be left unchanged.